tests/fe/fe_rannacher_turek_01.cc

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2015 by the deal.II authors
+// Copyright (C) 2015, 2017 by the deal.II authors
```

Comment:
I brought this up because I thought that the copyright script choked when it saw commas like this but now that I look back through the commit logs I can't seem to find any such problems. You are right, of course, that this is not worth worrying about.